cmd/hermes/cmd_eth.go

```diff
@@ -127,6 +129,14 @@ var cmdEthFlags = []cli.Flag{
 		Destination: &ethConfig.Libp2pPort,
 		DefaultText: "random",
 	},
+	&cli.IntFlag{
```

Comment:
you can make this a `time.Duration`:

```suggestion
	&cli.DurationFlag{
```